README.md

```diff
@@ -5,6 +5,9 @@
 ## PROJECT DESCRIPTION
 This project is about building a simple to-do list project.
 
+- Implemented a function for the "Clear all completed" button.
+- Stored updated array items i local storage 
+
```

Comment:
- [ ] **REQUIRED:** This PR should have all the changed files with the requirement met, not only the README file with some modification. It is not good practice to open a PR with only one commit. I supposed you have accidentally opened that PR, please consider having more commits or open another PR  with the link to the old PR and close that one. Also and cherry-pick all the commits in the new PR